.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Create a report to help us improve GNU Radio
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# 1. Have you looked in our documentation and asked the community?
+
+<!--- [Before you open an issue we suggest checking the following: -->
+<!--- - [The wiki](https://wiki.gnuradio.org/index.php/Main_Page) -->
+<!--- - [The GNU Radio Blocks documentation](https://wiki.gnuradio.org/index.php/Category:Block_Docs) -->
+<!--- - [The User manual](https://wiki.gnuradio.org/index.php/Usage_Manual) -->
+<!--- - [GNU Radio community](https://wiki.gnuradio.org/index.php/Chat) -->
+<!--- - or on [the mailing list](https://wiki.gnuradio.org/index.php/MailingLists)] -->
+
+# 2. GNU Radio version
+
+
+# 3. How did you install GNU Radio
+<!--- [Choose an option from this list -->
+<!--- - OS package manager -->
+<!--- - Built from source -->
+<!--- - PyBOMBS -->
+<!--- - Anaconda -->
+<!--- - Binary (please link to where you downloaded it) -->
+<!--- - Another way (Describe in 1-2 lines)] -->
+
+# 4. Can you provide a GRC to help us reproduce the issue
+<!-- [Drag and drop the .GRC file here.] -->
+
+# 5. What were you trying to do?
+<!-- [Give a 1-2 line explanation.] -->
+
+# 6. Expected Behavior
```

Comment:
Personally, "Expected" and "Actual Behaviour" are the heart of the actual report – it might be nice from a motivational point of view to let the user state them early on. 

From a working through bug reports point of view, it certainly would be nice if they were on top of the page.

Reply:
Also, encouraging a user to actually be precise here can be very helpful, especially in a math-y project.

For example 

```
What you expect to happen. Please be as precise as possible, it helps us find a solution!
```